index.js

```diff
@@ -21,7 +21,8 @@ function elevateCategory(group) {
         .filter(entry => !!entry)
```

Comment:
We can switch the order of these `map`/`filter`, so far we are iterating over the whole array twice as we are mapping and then filtering, if we switch them, we will only map the values that will be used.